spec/features/user_visits_book_show_page_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe 'user visits show page for specific book' do
+  before :each do
+    @user = User.create!(name: "Todd")
+    @book = Book.create!(title: "Narnia")
+    @review = Review.create!(body: "This book is great", author: @user.name, rating: "4", user: @user, book: @book)
```

Comment:
Again: take out author here. We have access to the user's name with something like `@review.user.name`.